File: trade-signal.jl

using GoldMinerTradeSignal
function main()

    parmfile = "https://iwasnothing.github.io/GoldMinerTradeSignal/GDX_parm.csv"
    
    msg = formatMsg(getSignal(parmfile))
    @show msg
    open("signal.html", "w") do io
        write(io, "<H1>" * msg)
    end;
end

main()

File: trade_signal.pl

using GoldMinerTradeSignal

parmfile = "https://iwasnothing.github.io/GoldMinerTradeSignal/GDX_parm.csv"

m = formatMsg(getSignal(parmfile))


open("results.html", "w") do io
    write(io, m)
end

ENV["RESULTS"] = m
ENV["RESULTS_FILE"] = "results.html"


